src/assets/i18n/it/translation.json

```
@@ -0,0 +1,97 @@
+{
+  "title": "Digital Green Certificate",
+  "imprint": "Impressum",
+  "data-privacy": "Privacy dei dati",
+  "record-patient-data": "Dati del paziente",
+  "record-result": "Risultati dei test",
+  "record-qr-scan": "Dati del QR code",
+  "welcome": "Benvenuti",
+  "process": "Processato",
+  "record-data": "Acquisizione dati",
+  "query-id-card": "Per favore esegui la ricerca con il numero della Carta d'Identitá!",
+  "clear": "Cancella il modulo",
+  "next": "Successivo",
+  "first-name": "Nome",
+  "standardised-first-name": "Nome Standard",
+  "name": "Cognome",
+  "standardised-name": "Cognome Standardised",
+  "date-of-birth": "Data di nascita",
+  "sex": "Sesso",
+  "male": "Maschio",
+  "female": "Femmina",
+  "other": "Altro",
+  "unknown": "Sconosciuto/a",
+  "day": "Giorno",
+  "month": "Mese",
+  "year": "Anno",
+  "address": "Indirizzo",
+  "zip": "Codice postale",
+  "city": "Città",
+  "street": "Via",
+  "house-number": "Numero Civico",
+  "phone-number": "Numero di Telefono",
+  "email-address": "E-Mail",
+  "test-id": "Test-ID*",
+  "processing-consent": "Consenti il processo nel CWA",
+  "bill-status": "stato della fattura",
+  "data-privacy-approve": "Approva il trattamento dei dati",
+  "patientdata-exclude": "Aggiungi i dati del paziente nel QR-Code",
+  "patient-data-correction": "Correggi i dati del paziente",
+  "process-finish": "Finisci il processo",
+  "qr-code": "QR Code",
+  "patient-data": "Dati del paziente",
+  "cancel": "Cancella",
+  "data-submit": "Inserisci i dati",
+  "result-positiv": "Positivo",
+  "result-negative": "Negativo",
+  "result-failed": "Fallito",
+  "process-number": "Pratica numero",
+  "logout": "Disconnetti",
+  "wrong-process-number": "Pratica numero errata",
+  "unknown-process-number": "Pratica numero {{processNo}}, sconosciuta",
+  "server-error": "Errore del sistema: {{status}}",
+  "server-not-reachable": "Il sistema non è raggiungibile",
+  "connection-error": "Errore di connessione: {{message}}",
+  "qr-scan": "Scannerizza il QR-Code",
+  "qr-scan-https-only": "QR-Scan funziona solo con il protocollo https",
+  "qr-code-no-patient-data": "Il codice QR-code non contiene dati del paziente",
+  "error-message": "Messaggio di errore",
+  "serverError": "É accaduto un errore:",
+  "error-patient-data-load": "Non siamo riusciti a caricare i dati del paziente.",
+  "statistics": "Statistiche",
+  "failed-report": "Impossibile caricare la pratica per il test",
```

Is this "a failure in producing a report of tests" or a "report of tests which had a failure"?
I tend to believe the second one given the English original.
```suggestion
  "failed-report": "Report dei test falliti",
```

---

Unfortunately this PR has being open for too long and the current reference files have diverge.
I believe this was meant to show an error message when the test couldn't be loaded.